pkg/blob/nodeserver_test.go

```diff
@@ -559,3 +570,21 @@ func TestNodeExpandVolume(t *testing.T) {
 		t.Errorf("Unexpected error: %v", err)
 	}
 }
+
+func TestMountUsingBlobfuseProxy(t *testing.T) {
+	args := "--tmp-path /tmp"
+	authEnv := []string{"username=blob", "authkey=blob"}
+	d := NewFakeDriver()
+	_, err := d.mountUsingBlobfuseProxy(args, authEnv)
+	// should be context.deadlineExceededError{} error
```

Comment:
remove comment?

Reply:
the return error is nil, right?

Reply:
it will always return an error, because we are passing fake arguments. So in the test case I am checking if there is an error then tests are passed.